RestartManager.js

```diff
@@ -0,0 +1,33 @@
+let NativeCodePush = require("react-native").NativeModules.CodePush;
+
+const RestartManager = (() => {
+    let _allowed = true;
+
+    function restartApp(onlyIfUpdateIsPending = false) {
+        if (_allowed) {
+            NativeCodePush.restartApp(onlyIfUpdateIsPending);
+        }
+    }
+
+    function allow() {
+        _allowed = true;
+        restartApp(true);
+    }
+
+    function allowed() {
```

Comment:
We're discussing removing this method altogether, so we can discuss the name if we decide to leave it in.